FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -122,6 +125,7 @@ - (instancetype)init:(RCNConfigFetch *)configFetch
 
     gMaxRetryCount = gMaxRetries;
     gIsRetrying = false;
+    gIsFeatureDisabled = false;
```

Comment:
would suggest naming this `gIsRealtimeDisabled`

Reply:
Updated.